custom/icds_reports/static/css/icds_dashboard.css

```diff
@@ -874,23 +874,11 @@ hr {
     vertical-align: middle;
```

Comment:
Why is their custom text alignment in this report?

Reply:
Right, I am going to remove it.